includes/specials/SpecialVersion.php

```diff
@@ -344,7 +344,7 @@ function getExtensionCredits() {
 		}
 
 		if ( count( $wgFileExtensions ) ) {
-			$out .= $this->openExtType('File extensions allowed for upload');
+			$out .= $this->openExtType( wfMsg( 'version-file-extensions-allowed' ) );
```

Comment:
I know the rest of the file is probably using the deprecated `wfMsg` methods, but can you switch this to the newer i18n method: `wfMessage( 'version-file-extensions-allowed' )->plain()`? `->plain()` is safe output format here since it is being escaped when it is output.